soh/soh/Enhancements/randomizer/hook_handlers.cpp

```diff
@@ -1825,6 +1848,78 @@ void ObjComb_RandomizerWait(ObjComb* objComb, PlayState* play) {
+    {{ SCENE_BACK_ALLEY_DAY, 447 },              RAND_INF_RICHARDS_HOUSE_UNLOCKED },
+    {{ SCENE_BACK_ALLEY_NIGHT, 447 },            RAND_INF_RICHARDS_HOUSE_UNLOCKED },
+    {{ SCENE_DOG_LADY_HOUSE, 447 },              RAND_INF_RICHARDS_HOUSE_UNLOCKED },
+    {{ SCENE_DOG_LADY_HOUSE, 447 },              RAND_INF_RICHARDS_HOUSE_UNLOCKED },
+    {{ SCENE_BACK_ALLEY_HOUSE, 447 },            RAND_INF_ALLEY_HOUSE_UNLOCKED },
+    {{ SCENE_BACK_ALLEY_DAY, 1665 },             RAND_INF_ALLEY_HOUSE_UNLOCKED },
+    {{ SCENE_BACK_ALLEY_NIGHT, 1471 },           RAND_INF_ALLEY_HOUSE_UNLOCKED },
+    {{ SCENE_KAKARIKO_VILLAGE, 6801 },           RAND_INF_KAK_BAZAAR_UNLOCKED }, // Adult Night
+    {{ SCENE_KAKARIKO_VILLAGE, 6591 },           RAND_INF_KAK_BAZAAR_UNLOCKED }, // Adult Day
+    {{ SCENE_KAKARIKO_VILLAGE, 6813 },           RAND_INF_KAK_BAZAAR_UNLOCKED }, // Child Day
+    {{ SCENE_KAKARIKO_VILLAGE, 6814 },           RAND_INF_KAK_BAZAAR_UNLOCKED }, // Child Night
+    {{ SCENE_KAKARIKO_VILLAGE, 8871 },           RAND_INF_KAK_POTION_SHOP_UNLOCKED }, // Child Day/Night Rear
+    {{ SCENE_KAKARIKO_VILLAGE, 8846 },           RAND_INF_KAK_POTION_SHOP_UNLOCKED }, // Adult Night Rear
+    {{ SCENE_KAKARIKO_VILLAGE, 8639 },           RAND_INF_KAK_POTION_SHOP_UNLOCKED }, // Adult Day Rear
+    {{ SCENE_KAKARIKO_VILLAGE, 7822 },           RAND_INF_KAK_POTION_SHOP_UNLOCKED }, // Adult Night
+    {{ SCENE_KAKARIKO_VILLAGE, 7615 },           RAND_INF_KAK_POTION_SHOP_UNLOCKED }, // Child Day/Night and Adult Day
+    {{ SCENE_KAKARIKO_VILLAGE, 2495 },           RAND_INF_BOSS_HOUSE_UNLOCKED },
+    {{ SCENE_KAKARIKO_CENTER_GUEST_HOUSE, 447 }, RAND_INF_BOSS_HOUSE_UNLOCKED },
+    {{ SCENE_KAKARIKO_VILLAGE, 3750 },           RAND_INF_GRANNYS_POTION_SHOP_UNLOCKED }, // Child
+    {{ SCENE_KAKARIKO_VILLAGE, 3519 },           RAND_INF_GRANNYS_POTION_SHOP_UNLOCKED }, // Adult
+    {{ SCENE_POTION_SHOP_GRANNY, 447 },          RAND_INF_GRANNYS_POTION_SHOP_UNLOCKED },
+    {{ SCENE_KAKARIKO_VILLAGE, 5567 },           RAND_INF_SKULLTULA_HOUSE_UNLOCKED },
+    {{ SCENE_HOUSE_OF_SKULLTULA, 447 },          RAND_INF_SKULLTULA_HOUSE_UNLOCKED },
+    {{ SCENE_KAKARIKO_VILLAGE, 1471 },           RAND_INF_IMPAS_HOUSE_UNLOCKED },
+    {{ SCENE_IMPAS_HOUSE, 447 },                 RAND_INF_IMPAS_HOUSE_UNLOCKED },
+    {{ SCENE_KAKARIKO_VILLAGE, 447 },            RAND_INF_WINDMILL_UNLOCKED },
+    {{ SCENE_WINDMILL_AND_DAMPES_GRAVE, 2495 },  RAND_INF_WINDMILL_UNLOCKED },
+    {{ SCENE_KAKARIKO_VILLAGE, 4543 },           RAND_INF_KAK_SHOOTING_GALLERY_UNLOCKED }, // Day
+    {{ SCENE_KAKARIKO_VILLAGE, 4751 },           RAND_INF_KAK_SHOOTING_GALLERY_UNLOCKED }, // Night
+    {{ SCENE_SHOOTING_GALLERY, 447 },            RAND_INF_KAK_SHOOTING_GALLERY_UNLOCKED },
+    {{ SCENE_GRAVEYARD, 645 },                   RAND_INF_DAMPES_HUT_UNLOCKED }, // Child Day
+    {{ SCENE_GRAVEYARD, 390 },                   RAND_INF_DAMPES_HUT_UNLOCKED }, // Child Evening
```

Comment:
For some reason, dampe's hut was unlocked be default at night as child. This needs fixing at some point but I won't block because i don't want to throw more logic conflicts at you because of it